dev-bin/release.sh

```diff
@@ -37,6 +37,8 @@ fi
 
 tag="v$version"
 
+perl -pi -e "s/(?<=version = \").+?(?=\")/$version/g"  pkg/geoipupdate/geoip_updater.go
```

Comment:
I think we'll want to commit this as well before creating the tag. Perhaps we could do it if we continue past line 46. It might be worth checking how the other release scripts we have do that (assuming there are cases we do) for consistency between them for future maintenance.

Reply:
Yes, I think it would be most expected that it create a commit with the change it makes, and push it, similar to that script: https://github.com/maxmind/minfraud-api-python/blob/main/dev-bin/release.sh#L40-L60